src/components/Table/hoc/withTableSelection/withTableSelection.tsx

```diff
@@ -97,27 +99,56 @@ export function withTableSelection<I extends TableDataItem, E extends {} = {}>(
                     size="l"
                     checked={checked}
                     disabled={disabled}
-                    onUpdate={handler}
+                    onChange={handler}
                     className={b('selection-checkbox')}
                 />
             );
         }
 
-        private handleCheckBoxChange = (id: string, selected: boolean) => {
-            const {selectedIds, onSelectionChange} = this.props;
+        private handleCheckBoxChange = (
```

Comment:
Better to rename to `*Update` since it's another handler

Reply:
[a981071](https://github.com/gravity-ui/uikit/pull/374/commits/a9810718d2d7d0d33b923ed3153bf9de3fcc2d3d)